packages/material-ui/src/FormControl/FormControl.test.js

```diff
@@ -64,13 +64,19 @@ describe('<FormControl />', () => {
     });
 
     it('should not be filled initially', () => {
-      const wrapper = mount(<FormControl />);
-      assert.strictEqual(getState(wrapper).filled, false);
+      mount(
+        <FormControl>
+          <TestComponent fn={context => assert.strictEqual(context.filled, false)} />
```

Comment:
Thoughts on using a TestComponent ?

Reply:
I think there's more value in an actual integration test but just for the migration it's probably fine.

I would prefer asserting on a spy in `fn` though. Otherwise the test would pass if the assertion is never reached.